src/components/InstallModalContent.js

```diff
@@ -1,36 +1,88 @@
-import * as React from 'react';
+import * as React from "react";
 
 export class InstallModalContent extends React.Component {
-
-  constructor (props) {
+  constructor(props) {
     super();
-    this.state = 
-      {value:'', copied:false};
-      
-    
+    this.state = { value: "", copied: false };
+  }
+
+  copyToClipboard = e => {
+    console.log("got it");
+    var textField = document.createElement("textarea");
+    textField.innerText = this.command();
+    document.body.appendChild(textField);
+    textField.select();
+    document.execCommand("copy");
+    textField.remove();
+    e.target.focus();
+    setTimeout(
```

Comment:
why this is needed?

Reply:
This schedules a call to `setState` in 5 seconds so that the "Copied" popover only shows for that amount of time